src/Microsoft.DotNet.Build.Tasks/PackageFiles/publishtest.targets

```diff
@@ -150,12 +153,12 @@
       </_SupplementalTestData>
     </ItemGroup>
     <ItemGroup>
-      <SupplementalTestDataTestDir Include="@(_SupplementalTestData->'$(TestPath)/%(DestinationDir)%(Filename)%(Extension)')" />
```

Comment:
After further investigation and discussion, we've decided to keep SupplementalTestData for the time being.  Fortunately, it looks like we may be able to remove the copy to the OutDir; as stated in https://github.com/dotnet/buildtools/pull/1052#discussion_r79700466, that copy was likely done just because we archived the OutDir, but now we archive the TestDir.  I'll try removing it and verify that it doesn't break any scenarios.  If that works, then I can remove the CopySupplementalData target and combine these items with `SourcesToCopyToTestDir`.